nilearn/plotting/img_plotting.py

```diff
@@ -37,6 +39,43 @@
 # Core, usage-agnostic functions
 
 
+def check_threshold(thr, data):
```

Comment:
One of the point of creating this function was to be able to use it for connectome plotting. I don't think you have done that yet.